test/server/test_fxa_auth.py

```diff
@@ -0,0 +1,78 @@
+import urlparse
+import urllib
+from clientlib import ScreenshotsClient
+from urlparse import urljoin
+import random
+import requests
+import time
+from requests import HTTPError
+
+# Hack to make this predictable:
+random.seed(0)
+
+
+def get_url(url, expect=None):
+    result = urllib.urlopen(url)
+    if expect and result.getcode() != expect:
+        raise Exception("GET {} returned status {}".format(url, result.getcode()))
+    return result.read()
+
+# test we can change expiration and delete shots made previously
+def test_owned_shot_with_fxa_auth():
+    first_user = ScreenshotsClient()
+    second_user = ScreenshotsClient()
```

Comment:
You might name `second_user` as `browser_only_user` or something to make it clear.